### PR TITLE
added server.js

### PR DESCRIPTION
**Pull request by gamezsal**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const  routes = require('./controllers/homeRoutes');
```
Comment (jorgeHernandez1) [inline server.js]: This should only point to ./controllers

File: server.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+//import sequelize 
+const sequelize = require('./config/connection');
+const { all } = require('./controllers/homeRoutes');
```
Comment (jorgeHernandez1) [inline server.js]: Not necessary

File: server.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const express = require('express');
```
Comment (jorgeHernandez1) [inline server.js:1]: Missing path, sessions and handlebars

File: server.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+//import sequelize 
+const sequelize = require('./config/connection');
+const { all } = require('./controllers/homeRoutes');
+
+const app = express();
+const PORT = process.env.PORT || 3001;
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
```
Comment (jorgeHernandez1) [inline server.js:36]: missing static reference to public folder

**Review by jorgeHernandez1 — CHANGES_REQUESTED** [review]
Please review comments

**Review by CHANCEEWELL — APPROVED** [review]

File: server.js (hunk 3 above)
Comment (gamezsal) [reply server.js:1]: I added the request suggestions

File: server.js (hunk 4 above)
Comment (gamezsal) [reply server.js:36]: added the express static

**Pull request closed without merging**